src/App.js

```diff
@@ -19,6 +23,12 @@ const ContentWrapper = ({children}) => {
     const theme = React.useMemo(
         () => createTheme({
                 palette: {
+                    primary: {
+                        // light: will be calculated from palette.primary.main,
+                        main: variables.primary,
+                        // dark: will be calculated from palette.primary.main,
+                        // contrastText: will be calculated to contrast with palette.primary.main
+                    },
                     type: isDarkMode ? 'dark' : 'light',
```

Comment:
@abeerzahid update this line and remove hard coded values dark | light

Reply:
@faheemsajjad65 This is for the theme color palette.